python/scripts/pupil-data-copy.py

```diff
@@ -0,0 +1,39 @@
+from pipeline import pupil
```

Comment:
can you make a little docstring comment at the top of this file saying what it does: "Bringing from old whatever whatever table to new xxx table" or something like that

Reply:
Should I just version this whole module instead and make a document on it?

Reply:
not sure what is the "whole module"? I thought a comment would be ok but if you wanna do something nelse go ahead

Reply:
https://github.com/cajal/pipeline/pull/385/commits/c28e9ecdc1fe2f1099b345e2b5124212a4989602

This is what i meant

Reply:
oh ok

Reply:
actually I may still add a docstring in this file pupil-data-copy.py just to say whta it does. Literally just a line so people can call help on it and see what it does or so they don't have to read teh code

Reply:
still unresolved

Reply:
added the docstring under pupil-data-copy.py specifying what this job did